jax/experimental/jax2tf/README.md

```diff
@@ -14,6 +14,7 @@ More involved examples, including using jax2tf with
 Flax models and their use with TensorFlow Hub and Keras, are described in the
 [examples directory](https://github.com/google/jax/blob/master/jax/experimental/jax2tf/examples/README.md).
 
+See also some [internal ongoing design discussions](go/jax2tf-doc).
```

Comment:
Maybe we don't want to link this? This is interpreted as a relative path in the repo.
```suggestion
See also some internal ongoing design discussions at `go/jax2tf-doc`.
```